pkg/api/server/register_containers.go

```diff
@@ -1023,7 +1023,7 @@ func (s *APIServer) RegisterContainersHandlers(r *mux.Router) error {
 	//   500:
 	//     $ref: "#/responses/InternalError"
 	r.HandleFunc(VersionedPath("/libpod/containers/{name}/wait"), APIHandler(s.Context, libpod.WaitContainer)).Methods(http.MethodPost)
-	// swagger:operation POST /libpod/containers/{name}/exists libpod containerExists
+	// swagger:operation GET /libpod/containers/{name}/exists libpod containerExists
```

Comment:
wait is a post -> https://docs.docker.com/engine/api/v1.40/#operation/ContainerWait

Reply:
This is a comment applying to `/exists`, not `/wait` ... can you re-check?